### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by madara-tribe** [madara-tribe/Pytorch-base-Unet#1]
<img width="391" alt="train_loss" src="https://user-images.githubusercontent.com/48679574/172000988-ae1cd6f6-778b-4260-92da-ea0751da43ca.png">
<img width="366" alt="valloss" src="https://user-images.githubusercontent.com/48679574/172000995-d0909a6f-1712-45a2-a814-15f087e23703.png">
